data_lang/pretty_test.txt

```diff
@@ -205,184 +284,173 @@ Input
+>         simple_primitives: [
+>             null, false, true
+>         ],
+>         numeric_primitives: [
+>             -123456789, 123.456789
+>         ],
+>         stringy_primitives: "string"
+>     },
+>     compounds: [
+>         [1, 2, 3],
+>         {dict: "ionary"}
 >     ],
->     [1, [2, [3, [4, [5, [6]]]]]],
->     [[[[[5], 4], 3], 2], 1]
->   ]
+>     "variety-pack": [
+>         null,
+>         [
+>             "Los",      "pollitos",
+>             "dicen",    "pío",
+>             "pío",      "pío"
+>         ],
+>         [1, [2, [3, [4, [5, [6]]]]]],
+>         [[[[[5], 4], 3], 2], 1]
+>     ]
 > }
 
-Width > 32
+Width > 36
 Expect
 > {
->   primitives: {
->     simple_primitives: [
->       null,
->       false,
->       true
->     ],
->     numeric_primitives: [
->       -123456789,
->       123.456789
->     ],
->     stringy_primitives: "string"
->   },
->   compounds: [
->     [1, 2, 3],
->     {dict: "ionary"}
->   ],
->   "variety-pack": [
->     null,
->     [
->       "Los",
->       "pollitos",
->       "dicen",
->       "pío",
->       "pío",
->       "pío"
->     ],
->     [
->       1,
->       [2, [3, [4, [5, [6]]]]]
+>     primitives: {
+>         simple_primitives: [
+>             null, false, true
+>         ],
+>         numeric_primitives: [
+>             -123456789, 123.456789
+>         ],
+>         stringy_primitives: "string"
+>     },
+>     compounds: [
+>         [1, 2, 3],
+>         {dict: "ionary"}
 >     ],
->     [[[[[5], 4], 3], 2], 1]
->   ]
+>     "variety-pack": [
+>         null,
+>         [
+>             "Los",      "pollitos",
+>             "dicen",    "pío",
+>             "pío",      "pío"
+>         ],
+>         [
+>             1,
+>             [2, [3, [4, [5, [6]]]]]
+>         ],
+>         [[[[[5], 4], 3], 2], 1]
+>     ]
 > }
 
-Width > 26
+Width > 30
 Expect
 > {
->   primitives: {
->     simple_primitives: [
->       null,
->       false,
->       true
->     ],
->     numeric_primitives: [
->       -123456789,
->       123.456789
->     ],
->     stringy_primitives: "string"
->   },
->   compounds: [
->     [1, 2, 3],
->     {dict: "ionary"}
->   ],
->   "variety-pack": [
->     null,
->     [
->       "Los",
->       "pollitos",
->       "dicen",
->       "pío",
->       "pío",
->       "pío"
->     ],
->     [
->       1,
->       [
->         2,
->         [3, [4, [5, [6]]]]
->       ]
+>     primitives: {
+>         simple_primitives: [
+>             null, false, true
+>         ],
+>         numeric_primitives: [
+>             -123456789,
+>             123.456789
+>         ],
+>         stringy_primitives: "string"
+>     },
+>     compounds: [
+>         [1, 2, 3],
+>         {dict: "ionary"}
 >     ],
->     [
->       [[[[5], 4], 3], 2],
->       1
+>     "variety-pack": [
+>         null,
+>         [
+>             "Los",
+>             "pollitos",
+>             "dicen",    "pío",
+>             "pío",      "pío"
+>         ],
+>         [
```

Comment:
This one seems to look weirder than it used to?  I wonder if having the rule only apply to "leaf" containers, i.e. containers that don't have any containers, would help

Although arguably you may want to format dicts like

```
{k: [1,2], k2: [3,4], ... }
```

etc